keyboards/mokey/mokey64/info.json

```
@@ -0,0 +1,81 @@
+{
+    "keyboard_name": "MOKEY64",
+    "url": "",
+    "maintainer": "qmk",
+    "width": 15,
+    "height": 5,
+    "layouts": {
+        "LAYOUT_60_ansi": {
```

```suggestion
        "LAYOUT_64_ansi": {
```
This is the well-known `64_ansi` layout; please rename the layout macros accordingly.